sdk/eventhub/Microsoft.Azure.EventHubs/tests/ServiceFabricProcessor/TestState.cs

```diff
@@ -102,7 +102,7 @@ public void VerifyNormalStartup(int maxRetries)
             int retries = 0;
             while (!this.Processor.IsOpened && !this.HasShutDown && (retries < maxRetries))
             {
-                Thread.Sleep(1000);
+                Thread.Sleep(5000);
```

Comment:
We should really try to find a better solution then adding additional sleep time, all that does is end up making our test runs take longer and still have reliability issues. /cc @mikeharder

Reply:
It appears the intention of these APIs is to block until a desired state is detected and fail after some timeout.  If so, these APIs should take `TimeSpan maxDuration` rather than `int maxRetries`, and they should poll as frequently as reasonable (say 100ms instead of 1000ms or 5000ms).  This gives the API sufficient time to pass reliably but ensures the API returns as quickly as possible.